config/unicorn.rb.example

```diff
@@ -1,24 +1,25 @@
-app_path = "/home/huginn/current"
+wd = "/home/huginn/huginn"
```

Comment:
I guess this can't be set by the .env file?  Maybe it could be `File.expand_path(File.join(File.dirname(__FILE__), '..'))`?